docs/clipboard-pickling/explainer.md

```diff
@@ -115,11 +115,53 @@ Sites can read or write pickled versions of sanitized formats, by providing the
 
-On Linux, ChromeOS, and Android, MIME types are often used (though Linux and ChromeOS don’t have strong recommendations regarding clipboard format naming conventions). Therefore, a MIME type `"custom/format"` will be transformed to `"application/web;type="custom/format""`. Note that the pickling prefix `"application/web;type="` precedes the transformed format name, which has quotes around the format name.
+Native apps will need to read the web custom format map and parse its payload as an alternative to `EnumFormats` on Windows to fetch the mapping of the custom MIME type to web custom format. It can then request for the web custom format corresponding to a particular MIME type of interest. This also helps in [Delayed Rendering](https://docs.microsoft.com/en-us/windows/win32/dataxchg/clipboard-operations#delayed-rendering) of formats on Windows where the native app just provides an indication to the clipboard that a format is available. When this format is requested by some other app, it renders the content on-demand.
+
+The web custom format map will have the below naming convention per platform:
+
+On Windows it will be inserted as `Web Custom Format Map`, on MacOS `com.web.custom.format.map` & On Linux/Android/CrOS etc `application/web;type="custom/formatmap"`.
+The payload in this format map will be of type JSON with the key representing the MIME type and the web custom format as the value.
+e.g. On Windows the web custom format map will have the below payload:
+```
+{
+
+  "text/html" : "Web Custom Format0",
+  "text/plain" : "Web Custom Format1",
+  "text/csv" : "Web Custom Format2"
+}
+```
+
+On MacOS
+```
+{
+
+  "text/html" : "com.web.custom.format0",
+  "text/plain" : "com.web.custom.format1",
+  "text/csv" : "com.web.custom.format2"
+}
+```
+
+On Linux, ChromeOS, and Android
+```
+{
+
+  "text/html" : "application/web;type="custom/format0",
+  "text/plain" : "application/web;type="custom/format1",
+  "text/csv" : "application/web;type="custom/format2"
+}
+```
+
+The value in the JSON payload mentioned above contains the actual payload of the custom MIME type. It will be serialized in terms of raw bytes and will have the below formatting naming convention:
+
+On MacOS (and iOS), clipboard formats are named using the UTI reverse-DNS naming convention. Therefore, a MIME type `"text/html"` will be transformed to `"com.web.custom.format0"`. Note that the pickling prefix `"com.web"` precedes the transformed format name `"custom.format0"`.
```

Comment:
nit: Should this be "com.web.custom.format.0" rather than "com.web.custom.format0"? Not sure, but just asking for clarification

Reply:
Same as above.

Reply:
I don't really mind either way. LGTM